src/gui/advancedsettings.cpp

```diff
@@ -175,6 +192,45 @@ void AdvancedSettings::updateCacheSpinSuffix(int value)
         spin_cache.setSuffix(tr(" MiB"));
 }
 
+void AdvancedSettings::updateInterfaceAddressCombo(int) {
+    //Try to get the currently selected interface name
```

Comment:
can you follow the comment style we use? a whitespace after `//` (this applies to every comment you added)